website/content/docs/configuration/server.mdx

```diff
@@ -58,9 +58,10 @@ server {
 - `enabled` `(bool: false)` - Specifies if this agent should run in server mode.
   All other server options depend on this value being set.
 
-- `enabled_schedulers` `(array<string>: [all])` - Specifies which sub-schedulers
-  this server will handle. This can be used to restrict the evaluations that
-  worker threads will dequeue for processing.
+- `enabled_schedulers` `(array<string>: ["service", "batch", "system",
+  "sysbatch"])` - Specifies which sub-schedulers this server handles. Use this
+  to restrict the evaluations that worker threads dequeue for
+  processing.
```

Comment:
Maybe we should show what the actual default value is and then explain how that gets interpreted:

```suggestion
- `enabled_schedulers` `(array<string>: [])` - Specifies which sub-schedulers this server 
  handles. Use this to restrict the evaluations that worker threads dequeue for
  processing. Nomad treats the empty default value as `["service", "batch", "system",
  "sysbatch"]`
```